cli/src/templates/packages/nativewindui/tabs/app/(tabs)/index.tsx.ejs

```diff
@@ -390,7 +391,7 @@ const COMPONENTS: ComponentItem[] = [
     },
   },
   <% } %>
-  <% if (props.stylingPackage?.options.selectedComponents.includes('selectable-text')) { %>
+  <% if (props.stylingPackage?.options.selectedComponents.includes('text')) { %>
```

Comment:
Depending on what we want to do we can keep the example code in the index files or we can import example components from the examples generated by nwui once its updated to export those components

Reply:
however the example code is actually up to date here as far as I can tell when comparing with the website

Reply:
for context:

We spoke about having an examples flag that would generate a nativewindui-examples folder that has the same examples from the website
the difference being from the examples that generate now

1. they generate in outside the nativewindui folder so its clear they are examples
2. they export the example component
3. the example file and component will all have example in the name

then I can import the example code instead of having three copies of the each example in the different navigation stacks

However this could actually be a follow up task potentially since the existing examples in the index file work, the only issue is that we get an examples folder being generated on top of the examples already in the index.